modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/Logging.scala

```diff
@@ -64,34 +63,29 @@ object Logging {
   private val dateFormatter: DateTimeFormatter =
     DateTimeFormatter.ofPattern("YYYY-MM-dd HH:mm:ss.SSS").withZone(ZoneId.systemDefault())
 
+  /**
+   * Format a message into a typical ERROR message
+   * WARNING: doesn't remove sensitive info
+   */
+  def mkMessage[F[_]: Clock: Functor, S: Show](isError: Boolean, message: S) =
+    Clock[F].instantNow.map { now => show"${ if (isError) "ERROR" else "INFO" } ${dateFormatter.format(now)}: $message" }
+
```

Comment:
I wonder if this commit should be part of this release, as this is will be completely refactored in observability release with slf4j

Reply:
I can make the change less drastic if it will help you with rebasing?

Reply:
That's alright, thank you